### PR TITLE
Replace Google+ by Discord

### PR DESCRIPTION
**Pull request by abcdefg30**
Keeping the svg art for Google+ as it is referenced by [this](https://github.com/OpenRA/OpenRAWeb/blob/cb46f4a0795a8d6ec82a7e233ebb5e22aa6cea87/content/news/release-20131223.md) news article from 2013.

### CODE REVIEW

File: content/community.html [hunk 1]
```diff
@@ -10,7 +10,7 @@ <h3>Getting in Touch</h3>
...
-<p>You can also try <a href="https://forum.openra.net" alt="Forums">our forum</a>, <a href="http://www.moddb.com/games/openra" alt="ModDB">ModDB page</a>, <a href="https://twitter.com/openRA" alt="Twitter">Twitter</a>, <a href="https://www.facebook.com/openra" alt="Facebook">Facebook</a>, <a href="https://plus.google.com/100332364931123881367/" alt="Google+">Google+</a> or <a href="https://www.reddit.com/r/openra/" alt="Reddit">Reddit</a>.</p>
+<p>You can also try <a href="https://forum.openra.net" alt="Forums">our forum</a>, <a href="http://www.moddb.com/games/openra" alt="ModDB">ModDB page</a>, <a href="https://twitter.com/openRA" alt="Twitter">Twitter</a>, <a href="https://www.facebook.com/openra" alt="Facebook">Facebook</a>, <a href="https://discord.gg/UcUztfr" alt="Discord server">Discord server</a> or <a href="https://www.reddit.com/r/openra/" alt="Reddit">Reddit</a>.</p>
```
Comment (matjaeck) [inline content/community.html]: Would prefer to use `Discord` instead of `Discord server` for the anchor text.

File: lib/openra.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@
   "twitter" => { name: "Twitter", url: "https://twitter.com/openRA" },
   "reddit" => { name: "Reddit", url: "https://www.reddit.com/r/openra" },
   "youtube" => { name: "Youtube", url: "https://www.youtube.com/channel/UCRoiPL1J4K1-EhQeNazrYig"},
-  "gplus" => { name: "Google+", url: "https://www.google.com/+OpenraNet" },
+  "discord" => { name: "Discord", url: "https://discord.gg/UcUztfr" },
```
Comment (pchote) [inline lib/openra.rb]: This breaks the colour-ordering of the icons.  IMO the best ordering now is to put discord between twitter and reddit, and then move moddb to the immediate left of steam.

**Comment by abcdefg30** [comment]
Updated: Imho the icon fits even better between Facebook and Twitter (on my display at least).
discordicon

**Comment by pchote** [comment]
Ok, fine.

**Pull request merged**